src/Servers/Kestrel/test/Interop.FunctionalTests/HttpClientHttp2InteropTests.cs

```diff
@@ -25,38 +25,27 @@ namespace Interop.FunctionalTests
     /// <summary>
     /// This tests interop with System.Net.Http.HttpClient (SocketHttpHandler) using HTTP/2 (H2 and H2C)
     /// </summary>
-    // Attributes are here to avoid testing http. Remove when https://github.com/dotnet/aspnetcore/issues/24902 is resolved.
-    [OSSkipCondition(OperatingSystems.MacOSX, SkipReason="https://github.com/dotnet/aspnetcore/issues/24902")]
-    [MinimumOSVersion(OperatingSystems.Windows, WindowsVersions.Win10, SkipReason="https://github.com/dotnet/aspnetcore/issues/24902")]
     public class HttpClientHttp2InteropTests : LoggedTest
```

Comment:
Question only: Should these tests be quarantined for now❔

Reply:
Famous last words: it should just work 😄